inc/category.class.php

```diff
@@ -118,20 +119,35 @@ public static function getCategoryTree($rootId = 0, $helpdeskHome = false) {
          AND `faqs`.`knowbaseitemcategories_id` <> '0'
       ) > 0";
 
-      $formCategory = new self();
       if ($rootId == 0) {
-         $items = $formCategory->find("`level`='1' AND ($where)", '`name`');
+         $query = "SELECT *
```

Comment:
Replacing something deprecated with something that will be deprecated soon is probably not the way to go... (even if I won't block this PR -- I would on core :p)

Reply:
yes, i completly agree, but actually, i can't assure compatibility with both versions.
next pass (for next major version) will permits to remove this crappy query